src/google-maps/map-info-window/map-info-window.ts

```diff
@@ -122,7 +122,12 @@ export class MapInfoWindow implements OnInit, OnDestroy {
     this._eventManager.destroy();
     this._destroy.next();
     this._destroy.complete();
-    this.close();
+
+    // No info window has been created on the server. To avoid errors on the server,
+    // we do not call `close` here as it expects an instantiated info window instance.
+    if (this._googleMap._isBrowser) {
```

Comment:
It might be better to check if the `this._infoWindow` is defined.

Reply:
I think I should just expand the comment to mention that also there is no google-map instance, so also `_assertInitialized` in `close` will fail.

Reply:
That's depending on information from outside the class though. If we check for the `_infoWindow`, we'd only be depending on the local state of the component.

Reply:
I would personally still prefer `isBrowser` as `infoWindow` requires context on what the `close` method does. I don't feel strong about it, so updated to `_infoWindow`.